### ISSUE TITLE
Add a way in the template to center images?

### ISSUE THREAD

**Issue by bparks13** [open-ephys/onix1-bonsai-docs#49]
Is there any way to modify the template or to add some CSS that will center images? Using the default format of `![Title](../link/to/image.png)` attempts to stretch the image from end to end, which makes it appear centered, but if we want to set a particular width (i.e. ,`![Title](../link/to/image.png){width=600px}`) then it will always be a left-aligned image. 

It is possible cheat a little bit and directly inject HTML into the markdown file, but this goes against the [philosophy of Markdown](https://stackoverflow.com/questions/12090472/how-do-i-center-an-image-in-the-readme-md-file-on-github/12091563#12091563). Adding HTML like this works

```
<p align="center">
  <img src="../link/to/image.png">
</p>
```

But this doesn't look very clean in the Markdown file, and also might not be properly rendered depending on what is serving the page.

My preferred method would be to add an `align="center"` parameter after the pixel width if possible (i.e., `![Title](../link/to/image.png){width=600px,align="center"}`), I think this keeps it as clean as possible while allowing for some customizability. However, I genuinely do not know if this is possible on the backend, hence the Question label.

**Comment by cjsha** [comment]
I like Markdown because it's super legible in its raw format while editing, but I'm also finding it super limited in flexibility when it comes to formatting the website. For example, #44 (your ask can be implemented doing something like I did in this PR btw). I also can't, for example, put text and an image side-by-side in standard Markdown afaik. I'm kinda feeling like we just should just start using HTML whenever we want more out of our .md files. We know how these .md files are going to be used - by docfx. As long as docfx processes it ok (it does), and it passes linkchecks ok (I'm pretty sure it will unless you're doing some funky stuff like I was a few weeks ago) that is. I'm not sure what benefit there is to keep our markdown compatible with some base-flavor easily-readable markdown ideal although that's how I felt when I started writing these docs. 

I'm so close to saying I'd prefer to work purely in HTML because of its super flexibility. rst, md, etc. it all gets converted to html for websites anyway. I think the problem is its readability, but it's not as big of a jump like going from c to asm.

With that said, I know jonnew's big on left-alignment. I don't really mind much either way
